### PR TITLE
New Crowdin updates

### PR DESCRIPTION
**Pull request by modxbot**

### CODE REVIEW

File: core/lexicon/country/tr.inc.php [hunk 1]
```diff
@@ -247,11 +247,13 @@
...
 $_country_lang["tj"] = 'Tacikistan'; // Tajikistan
-$_country_lang["tk"] = 'Tokelau'; // Tokelau
+$_country_lang["tk"] = 'Tokelau
+'; // Tokelau
 $_country_lang["tl"] = 'Doğu-Timör'; // Timor-Leste, ex: East Timor
 $_country_lang["tm"] = 'Türkmenistan'; // Turkmenistan
 $_country_lang["tn"] = 'Tunus'; // Tunisia
-$_country_lang["to"] = 'Tonga'; // Tonga
+$_country_lang["to"] = 'Tonga
+'; // Tonga
```
Comment (JoshuaLuckers) [inline core/lexicon/country/tr.inc.php]: I don't think this is supposed to happen.

File: core/lexicon/country/tr.inc.php [hunk 2]
```diff
@@ -247,11 +247,13 @@
 $_country_lang["tg"] = 'Togo'; // Togo
 $_country_lang["th"] = 'Tayland'; // Thailand
 $_country_lang["tj"] = 'Tacikistan'; // Tajikistan
-$_country_lang["tk"] = 'Tokelau'; // Tokelau
+$_country_lang["tk"] = 'Tokelau
+'; // Tokelau
```
Comment (JoshuaLuckers) [inline core/lexicon/country/tr.inc.php]: I don't think this is supposed to happen.

File: core/lexicon/country/tr.inc.php [hunk 3]
```diff
@@ -271,7 +273,8 @@
 $_country_lang["vg"] = 'Virgin Adaları (İngiliz)'; // Virgin Islands (British)
 $_country_lang["vi"] = 'Virgin Adaları (ABD)'; // Virgin Islands (U.S.)
 $_country_lang["vn"] = 'Vietnam'; // Viet Nam
-$_country_lang["vu"] = 'Vanuatu'; // Vanuatu
+$_country_lang["vu"] = 'Vanuatu
+'; // Vanuatu
```
Comment (JoshuaLuckers) [inline core/lexicon/country/tr.inc.php]: I don't think this is supposed to happen.

**Review by JoshuaLuckers — CHANGES_REQUESTED** [review]

File: core/lexicon/country/tr.inc.php (hunk 2 above)
Comment (Jako) [reply core/lexicon/country/tr.inc.php]: This issue should be created inside of Crowdin and has to be changed there.

Comment (JoshuaLuckers) [reply core/lexicon/country/tr.inc.php]: Who's able to do that? @matdave maybe?

Comment (Jako) [reply core/lexicon/country/tr.inc.php]: The MODX Crowdin Managers are @opengeek, @jaygilmore, @theboxer, @sepiariver

**Pull request merged**